ffi/zeromq_h.lua

```diff
@@ -109,4 +113,6 @@ int zpoller_add(zpoller_t *, void *);
 void *zpoller_wait(zpoller_t *, int);
 bool zpoller_expired(zpoller_t *);
 bool zpoller_terminated(zpoller_t *);
+int zmq_send(void *, void *, size_t, int);
+int zframe_send(zframe_t **, void *, int);
```

Comment:
Not the biggest deal but these seem to be in the "wrong" location compared to their definition?